CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Welcome to the Momento Go-Redis contributing guide :wave:
+
+Thank you for taking your time to contribute to our Momento `go-redis` wrapper!
+<br/>
+This guide will provide you information to start your own development and testing.
+<br/>
+Happy coding :dancer:
+<br/>
+
+## Requirements :coffee:
+
+- Go 1.18 or above is required https://go.dev/doc/install
+- A Momento auth token is required, you can generate one using the [Momento Console](https://console.gomomento.com)
+
+<br/>
+
+## Build :computer:
+
+```
+make build
+```
+
+<br/>
+
+## Formatting and Tidy :flashlight:
+
+```
+make lint
+```
+
+<br/>
+
+## Tests :zap:
+
+### Run integration tests against Momento 
+
+```bash
+TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
+make test-momento
+```
+### Run integration tests against Redis
+
+First run Redis either natively, run Redis in a Docker container, or do your development in a devcontainer. Here is an example of running Redis in a Docker container:
```

Comment:
I have no idea what a devcontainer is :-) I see you have a `.devcontainer` directory in the repo. Can you add some docs about how that works or a link to some docs?

Reply:
you're right. removed!